mergesort/mergesort.py

```diff
@@ -1,23 +1,29 @@
 
 __all__ = ["merge_sort"]
 
-def merge_sort(a,b,n,m):
+def merge_sort(a,b): 
     """
-    Creates a new ordonated succesion with all elements from two given ordonated series
+    Creates a new sorted succesion with all elements from two given sorted series
     
     Args:
         a: the first succesion
         b: the second succesion
-        n: the number of elements in a
-        m: the number of elements in b
     Returns:
         c: the created succesion using merge sort
     """
 
     c=[]
     i=0
     j=0
-    while i<n and j<m :
+    
+    if len(a)==len(b)==0:
+        return c
+    if len(a)==0: 
+        return b
+    if len(b)==0:
+        return a
+    
+    while i<len(a) and j<len(b) :
```

Comment:
len(a), len(b) get computed every time the check for the 'while' loop is performed; extract them in a var before the cycle